### PR TITLE
Added ESSID and BSSID fields, use floats for bitrate

### PR DESCRIPTION
**Pull request by abencz**
iwconfig in 12.04 is giving me a floating point value for bitrate. The other two additions are so we can monitor handoff between APs.

### CODE REVIEW

File: wireless_watcher/nodes/watcher_node [hunk 1]
```diff
@@ -17,16 +17,16 @@ previous_error = False
...
     if m:
-        return int(m.group(0))
+        return type(m.group(0))
 
 class Connection(wireless_msgs.msg.Connection):
     def __init__(self, fields):
         args = {}
 
-        args['bitrate'] = trim(fields['Bit Rate'])
+        args['bitrate'] = trim(fields['Bit Rate'], float)
```
Comment (abencz) [inline wireless_watcher/nodes/watcher_node]: I'm not sure this is the best way to do this. Added a parameter seems sloppy.

**Comment by mikepurvis** [comment]
Would prefer not to change the message hash. :(

On any modern connection, bits/s will be a pretty high number. Can you not simply round it?

**Comment by abencz** [comment]
Even if I round it, I still need the bssid/essid fields.

**Comment by mikepurvis** [comment]
Right, didn't realise that. Merge away.

**Pull request merged**